src/port1.0/portutil.tcl

```diff
@@ -3294,6 +3305,9 @@ proc _check_xcode_version {} {
             }
         }
         if {$xcodeversion eq "none"} {
+            if {[tbool use_xcode]} {
+                return -code error "This port requires Xcode, which was not found on your system."
+            }
             ui_warn "Xcode does not appear to be installed; most ports will likely fail to build."
             if {[file exists "/Applications/Install Xcode.app"]} {
                 ui_warn "You downloaded Xcode from the Mac App Store but didn't install it. Run \"Install Xcode\" in the /Applications folder."
```

Comment:
This message should still be shown in the case where use_xcode is true.

Reply:
Moved